planemo/shed_lint.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Yes, I wasted a good 10 minutes confused by what was happening, until I `import xml; print dir(xml)` and realised that python was importing `planemo.xml`.